services/grizzly/launch-grizzly.sh

```diff
@@ -61,20 +61,6 @@ EOF
 mkdir -p /var/lib/td-agent-bit/pos
 /opt/td-agent-bit/bin/td-agent-bit -c /etc/td-agent-bit/td-agent-bit.conf
 
-# shellcheck disable=SC2317
-function onexit () {
-  echo "Waiting for logs to flush..." >&2
-  sleep 15
-  killall -INT td-agent-bit || true
-  sleep 15
-  if [[ -d /tmp/grizzly ]]
-  then
-    echo "Saving grizzly logs..." >&2
-    cp -r /tmp/grizzly /logs/tmp-grizzly
-  fi
```

Comment:
Only these lines should be deleted. The waiting/killing is necessary or we lose the end of logs.